llvm/include/llvm/IR/Intrinsics.td

```diff
@@ -2103,6 +2105,12 @@ let IntrProperties = [IntrNoMem, IntrNoSync, IntrWillReturn] in {
                                LLVMMatchType<0>,
                                LLVMScalarOrSameVectorWidth<0, llvm_i1_ty>,
                                llvm_i32_ty]>;
+  def int_vp_clmul : DefaultAttrsIntrinsic<[ llvm_anyvector_ty ],
```

Comment:
Did you mean to add this intrinsic too? If so, it needs its own documentation, tests, etc.

Reply:
no. was just copying fshr code to try and get the vector version as well

Reply:
so to be clear,  should this line just get deleted?